src/main/java/edu/umich/tl/Course.java

```diff
@@ -0,0 +1,34 @@
+package edu.umich.tl;
+
+public class Course {
+	private int id;
+	private String courseName;
+	private String startDate;
```

Comment:
In `Term.java`, you use a date type for `startDate` and `endDate`.  Why use a string here?